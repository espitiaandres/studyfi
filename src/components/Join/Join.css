* {
  box-sizing: border-box;
  /* font-family: 'Segoe UI'; */
  font-family: 'Segoe UI', '-apple-system', 'HelveticaNeue', sans-serif;
}
  
.joinOuterContainer {
  align-items: center;
  display: flex;
  height: 100vh;
  justify-content: center;
  text-align: center;
}
  
.joinInnerContainer {
  width: 320px;
}
