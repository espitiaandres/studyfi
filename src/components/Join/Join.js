//
//  Join.js
//  react-spotify-player
//
//  Created by Andres Espitia.
//  Copyright Â© 2020 Andres Espitia. All rights reserved.
//

import React from 'react';
import './Join.css';
import JoinForm from '../JoinForm/JoinForm';

const Join = ({ duplicate }) => {
    
    return (
        <div>
        { 
            duplicate ? 
            <div>
                <div className="joinOuterContainer">
                    <div className="joinInnerContainer">
                        <p className="duplicatemessage">That username is already taken.</p> 
                        <p className="duplicatemessage">Please use another one!</p>
                        <JoinForm />
                    </div>
                </div>
            </div>
            : 
            <div className="joinOuterContainer">
                <div className="joinInnerContainer">
                    <JoinForm />
                </div>
            </div>
        }
        </div>

    )
}

export default Join;
