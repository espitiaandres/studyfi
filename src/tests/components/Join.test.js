//
//  Join.test.js
//  react-spotify-player
//
//  Created by Andres Espitia.
//  Copyright Â© 2020 Andres Espitia. All rights reserved.
//

import React from 'react';
import { shallow } from 'enzyme';
import Join from '../../components/Join/Join';

test('should render Join component correctly', () => {
    const wrapper = shallow(<Join />);
    expect(wrapper).toMatchSnapshot();
});
