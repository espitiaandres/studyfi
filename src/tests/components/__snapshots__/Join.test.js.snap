// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Join component correctly 1`] = `ShallowWrapper {}`;
